### DISCUSSION TITLE
Guacamole remote access software

### DISCUSSION THREAD

**Discussion by billgatos** [MichaIng/DietPi#6994] [Software Request]
Hi Daniel,

Being a fan of dietpi for severall years, i'm here requesting if possible to add teh Guacamole software to your project.
the soft is here https://guacamole.apache.org/
Best regards.
Roberto Rodrigues

**Comment by MichaIng** [comment]
Daniel (Fourdee) is not maintaining DietPi anymore, as you can see from the GitHub repo owner 😉.

Guacamole allows remote desktop access via web browser/HTTP, using either VNC, RDP or SSH as backend server protocol? Hence the use case is that you want to access your desktop (or console) from a system, where no VNC, RDP or SSH client is installed?

Given that there are tiny portable clients for all those protocols, there seems to be a rare need for this. Though Guacamole states to be faster  than default VNC connections by reducing the bandwidth. And there may be cases where default VNC/RDP/SSH ports may not be usable from the client network, but only HTTP/S.
